### ISSUE TITLE
Add tutorial for Nek-SAM coupling

### ISSUE THREAD

**Issue by aprilnovak** [neams-th-coe/cardinal#579]
A tutorial on the Nek-SAM separate domain coupling would greatly help new users learn how to leverage this part of Cardinal.

**Comment by aprilnovak** [comment]
@ahuxford what is the status of the double T-junction case you were working on? Do you think that could be made available as a tutorial?

**Comment by ahuxford** [comment]
Yes I have a tutorial started for it that I never finished up. I can finish it within the next week and make a pull request.

**Comment by aprilnovak** [comment]
That's great! And no rush - just wanted to make an issue to keep track of this.

**Comment by ahuxford** [comment]
I haven't forgotten about this and will add it ASAP. If there's any reason to hurry let me know and I can try to prioritize this more.

**Comment by aprilnovak** [comment]
No rush at all - take your time. I think as long as something goes in before Nureth that's fine (so that we can point people to a tutorial if they're interested).
